src/react-components/src/components/01-atoms/Button/Button.stories.tsx

```diff
@@ -8,6 +8,38 @@ export default {
   component: Button,
 };
 
-export const buttonFilledIcon = () => <Button callback={action("clicked")} id="button" content="search" type="filled" iconPosition="left" iconName="search-small" iconDecorative={true}/>;
+export const buttonFilledIcon = () =>
+  <Button
+    callback={action("clicked")}
+    id="button"
+    content="search"
+    type="filled"
+    iconPosition="left"
+    iconName="search-small"
+    iconDecorative={true}
+  />
+;
 export const buttonOutline = () => <Button callback={action("clicked")} id="button" content="Hello World" type="outline" />;
 export const buttonWithSpan = () => <Button callback={action("clicked")} id="button" type="outline" content={<span>Style my span!</span>}/>;
+export const buttonIconChildrenString = () =>
+  <Button
+    callback={action("clicked")}
+    id="button"
+    type="filled"
+    iconPosition="right"
+    iconName="search-small"
+    iconDecorative={true}
+  >Search!</Button>
+;
+export const buttonIconChildrenElement = () =>
```

Comment:
Might be not your wheelhouse but ... 
Who do we think stories are for?  Is it necessary for stories to have text as well as span, when visually they look the same?  (Is it confusing if they look the same visually?) . 
For engineers, is there a way we can document that children can be passed as either, the way that we can with props?

Reply:
I think it's fine so all the possible ways to render something are shown. Visually there's not much of a difference but you can look at the title of the story as well as the story's code. Ideally, developers would read the documentation on the component. In this case, it does say that either way can be passed:

![Screen Shot 2020-03-03 at 3 03 04 PM](https://user-images.githubusercontent.com/1280564/75814680-2b204480-5d60-11ea-957f-1c53fd5ffdeb.png)
![Screen Shot 2020-03-03 at 3 02 59 PM](https://user-images.githubusercontent.com/1280564/75814683-2b204480-5d60-11ea-9be4-b5a65546b9fb.png)

Reply:
I guess I was wondering if there was a way we can put the children type in docs the way we have it for proptypes ... 
I don't think it hurts to have them both here, but until we know better how people will use the storybook, it's also not clear what is useful and what isn't

Reply:
I see. It's just that `this.props.children` is prop that is always automatically there and used for pretty much any component. It's also almost never really described or documented the same way (as in a type or interface) since it's always expected. I think just seeing examples may suffice since most React devs can pickup how to use it.